test-app/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/integration/HmppsQueueSpyBeanTest.kt

```diff
@@ -30,45 +18,4 @@ class HmppsQueueSpyBeanTest : IntegrationTestBase() {
 
     verify(outboundSqsClientSpy).getQueueAttributes(any())
   }
-
-  @Test
-  fun `Can verify usage of spy bean for retry-dlq endpoint`() {
```

Comment:
I don't want to drop all of these tests as they were quite important.  I think there must be a better way - don't really understand why we had to drop the spy for the DLQ.

Reply:
ok will have a chat